easse/cli.py

```diff
@@ -118,98 +147,165 @@ def is_castable_to_float(obj):
+            lowercase=lowercase,
+        )
+
+    if "sent_bleu" in metrics:
+        metrics_scores["sent_bleu"] = corpus_averaged_sentence_bleu(
+            sys_sents, refs_sents, tokenizer=tokenizer, lowercase=lowercase
+        )
+
+    if "sari" in metrics:
+        metrics_scores["sari"] = corpus_sari(
+            orig_sents,
+            sys_sents,
+            refs_sents,
+            tokenizer=tokenizer,
+            lowercase=lowercase,
+        )
+
+    if "sari_legacy" in metrics:
+        metrics_scores["sari_legacy"] = corpus_sari(
+            orig_sents,
+            sys_sents,
+            refs_sents,
+            tokenizer=tokenizer,
+            lowercase=lowercase,
+            legacy=True,
+        )
+
+    if "sari_by_operation" in metrics:
+        (
+            metrics_scores["sari_add"],
+            metrics_scores["sari_keep"],
+            metrics_scores["sari_del"],
+        ) = get_corpus_sari_operation_scores(
+            orig_sents,
+            sys_sents,
+            refs_sents,
+            tokenizer=tokenizer,
+            lowercase=lowercase,
+        )
+
+    if "samsa" in metrics:
+        from easse.samsa import corpus_samsa
 
-    if 'sari_by_operation' in metrics:
-        metrics_scores['sari_add'], metrics_scores['sari_keep'], metrics_scores['sari_del'] = get_corpus_sari_operation_scores(orig_sents,
-                                                                                                                               sys_sents,
-                                                                                                                               refs_sents,
-                                                                                                                               tokenizer=tokenizer,
-                                                                                                                               lowercase=lowercase)
+        metrics_scores["samsa"] = corpus_samsa(
+            orig_sents,
+            sys_sents,
+            tokenizer=tokenizer,
+            lowercase=lowercase,
+            verbose=True,
+        )
 
-    if 'samsa' in metrics:
-        from easse.samsa import corpus_samsa
-        metrics_scores['samsa'] = corpus_samsa(orig_sents, sys_sents, tokenizer=tokenizer, lowercase=lowercase, verbose=True)
+    if "fkgl" in metrics:
+        metrics_scores["fkgl"] = corpus_fkgl(sys_sents, tokenizer=tokenizer)
 
-    if 'fkgl' in metrics:
-        metrics_scores['fkgl'] = corpus_fkgl(sys_sents, tokenizer=tokenizer)
+    if "f1_token" in metrics:
+        metrics_scores["f1_token"] = corpus_f1_token(
+            sys_sents, refs_sents, tokenizer=tokenizer, lowercase=lowercase
+        )
 
-    if 'f1_token' in metrics:
-        metrics_scores['f1_token'] = corpus_f1_token(sys_sents, refs_sents, tokenizer=tokenizer, lowercase=lowercase)
+    if "bertscore" in metrics:
+        metrics_scores["bertscore"] = corpus_bertscore(
```

Comment:
Doesn't this return 3 scores instead of one ? Shouldn't we keep only the f1 ?

Reply:
I think it's better to return the three values. I've modified it so that I behaves in a similar way to how we return the scores per SARI operation